### PR TITLE
Add prettier and pre-commit hook

### PR DESCRIPTION
**Pull request by paulojbleitao**

### CODE REVIEW

File: src/LoginScreen/LoginScreen.js [hunk 1]
```diff
@@ -66,161 +66,240 @@ class LoginScreen extends Component {
...
+                    className="font-white"
+                  >
+                    {this.state.username}
+                  </span>
                   <Input.Group compact>
-                    { oPasswordInput }
+                    {oPasswordInput}
                     <Button
                       onClick={() => this.submitPass(this.state.password)}
-                      style={{ width: '15%' }} type="primary"><Icon type="right" /></Button>
```
Comment (felipemarinho97) [inline src/LoginScreen/LoginScreen.js]: Can you explain why this is being removed?

File: src/messages.js [hunk 2]
```diff
@@ -1,263 +1,262 @@
...
-      Lock: "Заблокувати",
+      Hibernate: 'Сплячий режим',
+      Suspend: 'Згорнути',
+      Restart: 'Перезавантажити',
+      Shutdown: 'Вимкнути',
+      Lock: 'Заблокувати',
       UserPlaceholder: "Ім'я користувача",
-      PasswordPlaceholder: "Пароль",
+      PasswordPlaceholder: 'Пароль',
       UsernameWarning: "Будь ласка, введіть ім'я користувача.",
```
Comment (felipemarinho97) [inline src/messages.js]: Maybe prettier skipped this line? o.O

**Comment by felipemarinho97** [comment]
Such a prettier Pull Request! Just check out to the comments that i've made.

File: src/LoginScreen/LoginScreen.js (hunk 1 above)
Comment (paulojbleitao) [reply src/LoginScreen/LoginScreen.js]: It isn't, it's just being split into several lines for readability

File: src/messages.js (hunk 2 above)
Comment (paulojbleitao) [reply src/messages.js]: That's weird, I'll double check

Comment (paulojbleitao) [reply src/messages.js]: Actually, it's meant to be that way because there's an apostrophe inside the string

**Pull request merged**